packages/roosterjs-editor-dom/lib/utils/preprocessTable.ts

```diff
@@ -0,0 +1,47 @@
+import normalizeRect from './normalizeRect';
```

Comment:
If this function is only used by TableResize, put it under the table resize folder and no need to export from index.